docs/UserGuide.md

```diff
@@ -6,13 +6,19 @@
 
 # Avengers Assemble User Guide
 
-Avengers Assemble (AA) is a **desktop app for managing contacts, meant for use with a Command Line Interface** (CLI) 
+Avengers Assemble (AA) is a **desktop app for managing contacts**, meant for use with a Command Line Interface (CLI) 
 while still having the benefits of a Graphical User Interface (GUI). 
 
-The application is designed for Head Tutors of the NUS CS1101S Programming Methodology course, who intend to simplify their administrative tasks relating to contact management between students, other teaching assistants, and course instructors.
-However, its use cases can be extended to Head Tutors of other courses outside CS1101S. 
+The application is designed for **Head Tutors** of the NUS CS1101S Programming Methodology course, who intend to simplify their administrative tasks relating to contact management between students, other teaching assistants, and course instructors.
+However, its functions can also be extended to you if you are a head tutor of another course. 
 
-Our user guide aims to aid users in understanding how to get started with the application.
+Our user guide aims to help you understand how to get started with the application.
+
+
+We will cover the features in a chronological order, from:
+1. Installation, to 
+2. Simple commands like adding and editing, followed by 
+3. More advanced commands like filtering and exporting data.
```

Comment:
Good inclusion

Reply:
Nice our aim for UG is much detailed now